auth-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+info:
+  component: Authentication Service
+
+spring:
+  application:
+     name: AUTH-SERVICE
+  freemarker:
+    enabled: false
+  thymeleaf:
+    cache: false
+
+server:
+  context-path: /uaa
+  port: 8085
+
+eureka:
+  client:
+    serviceUrl:
+      defaultZone: http://127.0.0.1:8080/eureka/
```

Comment:
lets use the environment variable with default value to have the ability to customize this property